blog/content/post/2020-02-18-ew2020-btg-demo.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Boot Guard - pre-execution firmware verification
+abstract: 'This post will not describe how to guard your shoes. However,
+          will definitely introduce you to Boot Guard feature present on
+          Intel processors which allows firmware verification before the first
+          instruction executes. One may call it pre-execution firmware
+          verification.'
```

Comment:
@miczyg1 Staszek will use that description for social media so maybe we should mention coreboot, open source firmware and other keywords?

Reply:
@miczyg1 also in title we should mention Protectli as well as in description

Reply:
Done